rules/windows/process_creation/win_susp_use_of_sqltoolsps_bin.yml

```diff
@@ -0,0 +1,32 @@
+title: SQL Client Tools PowerShell Session Detection
+id: a746c9b8-a2fb-4ee5-a428-92bee9e99060 
+status: experimental
+description: PowerShell execution through builtin SQL Server Management Studio "SQLToolsPS.exe" binary. Microsoft PS logging like
+  ScriptBlock logging function of PowerShell is not an option here, PS session caused by the binary won't be recorded/logged.
+references:
+  - https://github.com/LOLBAS-Project/LOLBAS/blob/master/yml/OtherMSBinaries/Sqltoolsps.yml 
+  - https://twitter.com/pabraeken/status/993298228840992768 
+tags:
+    - attack.execution
+    - attack.t1059.001
+    - attack.defense_evasion 
+    - attack.t1127
+author: Agro (@agro_sev)
```

Comment:
could you also please mention `oscd.community` here in the author field? 
it will help with promotion and will engage more people for the next sprints. 
thank you!